src/main/java/com/github/dockerjava/core/DefaultDockerClientConfig.java

```diff
@@ -279,7 +279,7 @@ public AuthConfig effectiveAuthConfig(String imageName) {
 
     @Override
     public AuthConfigurations getAuthConfigurations() {
-        File dockerCfgFile = new File(getDockerConfig() + File.separator + DOCKER_CFG);
+        File dockerCfgFile = new File(getDockerConfig());
```

Comment:
The idea was that variable has path to docker config and adds DOCKER_CFG (that should be removed if it not used). But since what version all this things changed?